tests/CppWinRTAuthoringTests.cpp

```diff
@@ -21,6 +23,18 @@ struct my_async_status : winrt::implements<my_async_status, winrt::Windows::Foun
     void Close() {};
 };
 
+// This type has a settable property
+struct my_pointer_args : winrt::implements<my_pointer_args, winrt::Windows::UI::Xaml::Input::PointerRoutedEventArgs>
+{
+    wil::single_threaded_rw_property<bool> Handled{ false };

```

Comment:
Note: this did not need to be an `rw` property---it compiled fine with a normal property.

FYI @asklar @jonwis

Reply:
that's weird... if you make it non-RW, what gets called when you do `prop.Handled(true);` ?

Reply:
Per the mail thread on this, the `single_threaded_property` has an `operator()(T&&)` (set from C++/WinRT binding) and `operator=(T&&)` (set from normal C++ code) on it. I think it's probably OK to remove the `op()` on the non-RW version, but it could be a source-breaking change. That's also OK, as it's trivial to notice it and go "oh yeah I should be using `foo->Handled = true` like normal code and not `foo->Handled(true)`.